hkmc2Benchmarks/src/test/bench/LazySpreads/LazyArrayTest.mls

```diff
@@ -476,5 +460,5 @@ suite.add("sillyEager15", () =>
 //│ = [object Object]
 
 :silent
-suite |> runSuite
-writeResults(suite, "src/test/bench/LazySpreads/LazyArrayTest.json")
+// suite |> runSuite
```

Comment:
No longer running it?

Reply:
commited accidentally, about to revert :sweat_smile: